src/components/Dropdown/__tests__/__snapshots__/Dropdown.test.tsx.snap

```diff
@@ -2779,6 +2779,11 @@ exports[`Dropdown renders a value when given a matching option id through props
   pointer-events: none;
 }
 
+.c8 {
+  height: 1.125rem;
+  z-index: 1;
+}
+
```

Comment:
Are the changes to the snapshots expected?

Reply:
No they are not. Any idea why the changes would happen? I haven't touched any of the CSS to my knowledge.

Reply:
No, but adding an onClick where it is expected to be null could cause some conditional CSS to get inserted.